### DISCUSSION TITLE
How to add a docker registry endpoint with self signed certificat for replication

### DISCUSSION THREAD

**Discussion by hydezhao** [goharbor/harbor#20628] [Q&A] [answered]
How to add a docker registry endpoint with self signed certificat for replication

**Comment by MinerYang** [comment, marked as answer]
While your cert it self-signed, please uncheck the `Verify Remote Cert` box while creating registry.

**Comment by hydezhao** [comment]
actually we can put the ca file in `./common/config/shared/trust-certificates` and restart container. There is an `install_cert.sh` called by `entrypoint.sh` installing the cert to `/etc/pki/tls/certs/ca-bundle.crt`.
